app/screens/EditParentDetails.tsx

```diff
@@ -239,23 +243,23 @@ const EditParentDetails = ({ route, navigation }: Props): any => {
                 }
               }}
               value={parentName}
-              placeholder={t('parentPlaceNameTxt')}
-              placeholderTextColor={"gray"}
+              //placeholder={t('parentPlaceNameTxt')}
+              //placeholderTextColor={"#77777779"}

```

Comment:
I see these comments in many files. Let it be if you think you may need them, otherwise they should be removed.

Reply:
@Akhror Let's put this comment once testing done and we confirm the changes then remove this placeholder code.